### PR TITLE
Store: Move all shipping label dialog buttons to fixed footer

### PR DESCRIPTION
**Pull request by dechov**
Passes `buttons` prop to `<Dialog>` in order to keep buttons in a fixed footer, as requested by @kellychoffman in https://github.com/Automattic/wp-calypso/pull/19095#issuecomment-340000308.

Before | After
-- | --
 | 
 | 
 | 
 | 
 |

### CODE REVIEW

**Comment by kellychoffman** [comment]
Awesome! Can we just remove the `hr` on the Request a Refund modal? Its a bit much now that there's a border above the action buttons.

**Comment by dechov** [comment]
@kellychoffman Done:

**Comment by dechov** [comment]
Note: sort of out of scope for this PR, but I changed the details dialog heading to title case to match the order details heading — the label purchase & rest of these dialogs are sentence case though 🤔

**Comment by kellychoffman** [comment]
We should probably have them all be sentence case.

File: client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/packages-step/move-item.js [hunk 1]
```diff
@@ -104,11 +103,22 @@ const MoveItemDialog = ( props ) => {
 
 	const onClose = () => props.closeItemMove( orderId, siteId );
 
+	const buttons = [
+		{ label: translate( 'Cancel' ) },
```
Comment (DanReyLop) [inline client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/packages-step/move-item.js]: Do you think it's worth it to add `onClick: onClose` to this button (and the other `Close` buttons in the dialogs), just to be more explicit?

**Pull request merged**